packages/excalidraw/components/SearchMenu.tsx

```diff
@@ -154,16 +155,22 @@ export const SearchMenu = () => {
       const match = searchMatches.items[focusIndex];
 
       if (match) {
+        const zoomValue = app.state.zoom.value;
+
         const matchAsElement = newTextElement({
           text: match.searchQuery,
           x: match.textElement.x + (match.matchedLines[0]?.offsetX ?? 0),
           y: match.textElement.y + (match.matchedLines[0]?.offsetY ?? 0),
           width: match.matchedLines[0]?.width,
           height: match.matchedLines[0]?.height,
+          fontSize: match.textElement.fontSize,
```

Comment:
@ryan-di I believe this was an omission. Not including fontSize will give us incorrect width/height metrics.

Reply:
I don't think `scrollToContent`, where `matchAsElement` is used, measures the text again and the `width` and `height` provided do take the correct text measures. 

If we were to include `fontSize`, we should include `fontFamily` too.

Reply:
> I don't think `scrollToContent`, where `matchAsElement` is used, measures the text again and the `width` and `height` provided do take the correct text measures.

No, but we're using the dimensions of `matchAsElement` to scroll to, and the dimensions aren't correct.

> If we were to include `fontSize`, we should include `fontFamily` too.

good point